frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/DropDownSelect.jsx

```diff
@@ -24,40 +24,35 @@ const DropDownSelect = ({
   showPlaceHolder = false,
   highlightSelected = true,
   buttonClasses = '',
-  foreignKeyAccess = false,
-  showRedirection = false,
+
+  // from ToolJet database dashboard
+  tjdb = {
```

Comment:
@S-Abdul-Rahman 
1. Which all components are these used finally used in?
2. If we were to to state what functionality does these props help us enable, what will that be?